systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -153,4 +156,79 @@ public static void waitForClusterStability(String clusterName) {
             return false;
         });
     }
+
+    /**
+     * Method which, update/replace Kafka configuration
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param brokerConfigName key of specific property
+     * @param value value of specific property
+     */
+    public static void updateSpecificConfiguration(String clusterName, String brokerConfigName, Object value) {
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> {
+            LOGGER.info("Kafka config before updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+            Map<String, Object> config = kafka.getSpec().getKafka().getConfig();
+            config.put(brokerConfigName, value);
+            kafka.getSpec().getKafka().setConfig(config);
+            LOGGER.info("Kafka config after updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+        });
+    }
+
+    /**
+     * Method which, extends the @see updateConfiguration(String clusterName, KafkaConfiguration kafkaConfiguration, Object value) method
+     * with stability and ensures after update of Kafka resource there will be not rolling update
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param brokerConfigName key of specific property
+     * @param value value of specific property
+     */
+    public static void  updateConfigurationWithStabilityWait(String clusterName, String brokerConfigName, Object value) {
+        updateSpecificConfiguration(clusterName, brokerConfigName, value);
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.kafkaStatefulSetName(clusterName));
+    }
+
+    /**
+     * Verifies that updated configuration was successfully changed inside Kafka CR
+     * @param brokerConfigName key of specific property
+     * @param value value of specific property
+     */
+    public static boolean verifyCrDynamicConfiguration(String clusterName, String brokerConfigName, Object value) {
```

Comment:
What does this tell you that the result of the update to the `Kafka` CR does not?